src/Account.js

```diff
@@ -0,0 +1,131 @@
+import AGITokenAbi from 'singularitynet-token-contracts/abi/SingularityNetToken';
+import AGITokenNetworks from 'singularitynet-token-contracts/networks/SingularityNetToken';
+import Tx from 'ethereumjs-tx';
+
+export default class Account {
+  constructor(web3, networkId, config, mpeContract) {
+    const account = web3.eth.accounts.privateKeyToAccount(config.privateKey);
+    web3.eth.accounts.wallet.add(account);
+    this._web3Account = account;
+    this._web3 = web3;
+    this._config = config;
+    this._networkId = networkId;
+    this._tokenContract = this._generateTokenContract();
+    this._mpeContract = mpeContract;
+  }
+
+  async balance() {
+    return this._getTokenContract().methods.balanceOf(this.address).call();
+  }
+
+  async escrowBalance() {
+    return this._mpeContract.balance(this.address);
+  }
+
+  async depositToEscrowAccount(amountInCogs) {
```

Comment:
Its worth adding a check to see if the amount deposited has already been approved and approve only when required. Crypto aware developers would likely approve a big number upfront to prevent having to pay additional gas costs for approval at each deposit  step

Reply:
Makes sense. Will add a check here before moving tokens.

Reply:
Addressed in [commit](https://github.com/singnet/snet-sdk-js/pull/1/commits/807b13eaf7594dd6f2474117d2f1fa86edb80381)